client/homebrew/pages/basePages/listPage/listPage.jsx

```diff
@@ -136,13 +137,37 @@ const ListPage = createClass({
 		return;
 	},
 
-	updateUrl : function(filterTerm, sortType, sortDir){
+	updateListFilter : function(type, term){
+		this.updateUrl(this.state.filterString, this.state.sortType, this.state.sortDir, term);
+	},
```

Comment:
This appears to be a function that just calls another function. Is there a way we could just use `updateUrl` directly?

Reply:
This function is passed down to the BrewItem (line 86) to allow the tag onClick to update the URL while maintaining the state of the rest of the current filtering parameters. That said, the `type` parameter is never actually used - I had originally envisioned being able to set multiple parameters using this function, but that's not how I ended up actually making the code functional.
I'll take a deeper dive into it in a few hours time, once I'm free from work.

Reply:
What i meant, is can we pass down `updateUrl` instead of passing down this other function?

Reply:
Almost certainly, I'll see what I can do after work tonight.

Reply:
Latest commit should have the call directly to `updateUrl`.